src/data-access/repositories/interfaces/action-repository.ts

```diff
@@ -3,4 +3,5 @@ import { Action } from '../../../model/entities/action'
 export interface ActionRepository {
   getAction(actionId: string): Promise<Action>
   saveActions(actions: Action[]): Promise<void>
+  deleteRundownActions(rundownId: string): Promise<void>
```

Comment:
This makes it sound like we have a concept called `RundownAction` which we technically don't.
`deleteActionsForRundown` has a nicer ring to it.

Reply:
Renamed as suggested.